cdc/entry/schema/snapshot.go

```diff
@@ -139,6 +144,10 @@ func NewSnapshotFromMeta(meta *timeta.Meta, currentTs uint64, forceReplicate boo
 	tag := negative(currentTs)
 
 	for _, dbinfo := range dbinfos {
+		if filter.ShouldIgnoreSchema(dbinfo.Name.O) {
+			log.Info("ignore database", zap.String("db", dbinfo.Name.O))
```

Comment:
Will this log too much, ditto for L165, if there exist many schemas or tables

Reply:
It may be so, I will change it to debug level.